config/locales/javascript/javascript.en.yml

```diff
@@ -300,5 +300,8 @@ en:
       count:
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        one: "1 vote"
```

Comment:
I still have a bug here, looks like the "0" case is using the "one" translation. Any idea what's going on?